src/pages/Filing/FilingHome.tsx

```diff
@@ -104,8 +104,12 @@ function Home(): ReactElement {
                 >
                   Find answers to frequently asked questions
                 </Link>,
-                <Link href={sblHelpLink} key='ask-a-question' type='list'>
-                  Contact our support staff
+                <Link
+                  href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Error page: encountered an error on the SBL Platform'
```

Comment:
Link is on Filing homepage:
```suggestion
                  href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Filing home page: Seeking guidance'
```

Reply:
Not sure how I missed that, good catch!

Reply:
I'm also fan of "Seeking guidance" @meissadia 

![Screenshot 2024-03-29 at 2 36 53 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/0d776412-75ec-440f-a571-7a7c28e07b8a)